olca-app/openLCA.product

```diff
@@ -41,12 +41,12 @@ project and other open source software.
       <macosx icon="/olca-app/icons/logo/logo.icns"/>
       <win useIco="false">
          <bmp
-            winSmallHigh="/olca-app/icons/logo/16_32bit.bmp"
-            winSmallLow="/olca-app/icons/logo/16_8bit.bmp"
-            winMediumHigh="/olca-app/icons/logo/32_32bit.bmp"
-            winMediumLow="/olca-app/icons/logo/32_8bit.bmp"
-            winLargeHigh="/olca-app/icons/logo/48_32bit.bmp"
-            winLargeLow="/olca-app/icons/logo/48_8bit.bmp"/>
+            winSmallHigh="/olca-app/icons/logo/logo-16x16-32.bmp"
+            winSmallLow="/olca-app/icons/logo/logo-16x16-8.bmp"
+            winMediumHigh="/olca-app/icons/logo/logo-32x32-32.bmp"
+            winMediumLow="/olca-app/icons/logo/logo-32x32-8.bmp"
+            winLargeHigh="/olca-app/icons/logo/logo-48x48-32.bmp"
+            winLargeLow="/olca-app/icons/logo/logo-48x48-8.bmp"/>
```

Comment:
@msrocka it is pretty weird that on windows the icon being rendered was of eclipse because I can confirm that the icons that replaced were all openLCA icons and they were also being pointed correctly. There was one 256x256 icon missing that I added additionally.

Reply:
Not sure if this would fix it? It is possible to provide a single icon file here instead of multiple bitmaps. I thought that we build such a new single icon file that works for all image sizes on modern Windows.

Reply:
I can try that as well .

Reply:
@msrocka I think .ico should work for all . But I could not find a converter online. Could you share if you know of something. 
Before that should we try the above changes in windows ?